crates/bevy_sprite/src/sprite.rs

```diff
@@ -1,16 +1,22 @@
+use bevy_asset::Handle;
 use bevy_color::Color;
 use bevy_ecs::{component::Component, reflect::ReflectComponent};
 use bevy_math::{Rect, Vec2};
 use bevy_reflect::{std_traits::ReflectDefault, Reflect};
+use bevy_render::{texture::Image, view::Visibility, world_sync::SyncToRenderWorld};
+use bevy_transform::components::Transform;
 
-use crate::TextureSlicer;
+use crate::{TextureAtlas, TextureSlicer};
 
-/// Specifies the rendering properties of a sprite.
-///
-/// This is commonly used as a component within [`SpriteBundle`](crate::bundle::SpriteBundle).
+/// Describes a sprite to be rendered to a 2D camera
 #[derive(Component, Debug, Default, Clone, Reflect)]
+#[require(Transform, Visibility, SyncToRenderWorld)]
 #[reflect(Component, Default, Debug)]
 pub struct Sprite {
+    /// The image used to render the sprite
+    pub image: Handle<Image>,
+    /// The (optional) texture atlas used to render the sprite
```

Comment:
I think this should be `texture_atlas` instead of `atlas`. I prefer a 1:1 concept-to-field name here (and in most other places).